components/04-views/product-view/_product-view.scss

```diff
@@ -748,7 +753,7 @@ $product-view__grouped-product-name-margin\@medium     : 0 0 $spacer !default;
 
     &__grid-item {
         .catalog-grid-item__name {
-            font-size: 12px;
+            font-size: 16px;
```

Comment:
Why don't we use variable for font-size here? I see that there is no var for 16px, but there is no `$font-size-medium` we can assign this number to. What do you think?

Reply:
Yeah. We can do that.

Added:
```scss
$product-view__slider-name-font-size                   : $font-size-medium !default;
```

and
```
$font-size-medium           : 16px !default;
```